web/src/components/feature/ComponentTable.tsx

```diff
@@ -0,0 +1,50 @@
+import { EdsProvider, Table, TextField } from '@equinor/eds-core-react'
+
+export type TComponentEntry = {
+  componentId: string
+  altName: string
+  chemicalFormula: string
+}
+
+export const ComponentTable = ({
+  input,
+}: {
+  input: Array<TComponentEntry>
+}): JSX.Element => {
+  function createTableRows() {
+    return input.map((entry, index) => (
+      <Table.Row key={index}>
+        <Table.Cell
+          data-testid={`Component-${index}`}
+          key={`Component-${index}`}
+        >{`${entry.altName} (${entry.chemicalFormula})`}</Table.Cell>
+        <Table.Cell
+          data-testid={`Feed Value (mol)-${index}`}
+          key={`Feed Value (mol)-${index}`}
+        >
+          <TextField
+            id={`Feed Value (mol)-${index}-input`}
+            placeholder={'0'}
+            type="number"
+          />
+        </Table.Cell>
+      </Table.Row>
+    ))
```

Comment:
Kind of strange having a function that only has another function and calls that.
I would rather write ComponentTable like this.
```tsx
const ComponentTable = (...) {
return <>
{input.map(() =>....)}
</>
}
```

Reply:
Agree. Slip-up.

Reply:
Not possible.